README.md

```diff
@@ -167,11 +173,29 @@ if ($client->getOutcome() == \Yoti\YotiClient::OUTCOME_SUCCESS) {
 Where `yourUserSearchMethod` is a piece of logic in your app that is supposed to find a user, given a userId.
 No matter if the user is a new or an existing one, Yoti will always provide her/his profile, so you don't necessarily need to store it.
 
-The `ActivityDetails` class provides a set of methods to retrieve different user attributes. Whether the attributes are present or not depends on the settings you have applied to your app on Yoti Dashboard.
+The `profile` object provides a set of attributes corresponding to user attributes. Whether the attributes are present or not depends on the settings you have applied to your app on Yoti Dashboard.
+
+You can retrieve the sources and verifiers for each attribute as follows:
 
-### ActivityDetails
+```php
+<?php 
+$givenNamesSources = $profile->getGivenNames()->getSources(); // list or array of anchors
+$givenNamesVerifiers = $profile->getGivenNames()->getVerifiers(); // list or array of anchors
+```
 
-The set of attributes the user has configured for the transaction.
+You can also retrieve further properties from these respective anchors in the following way:
+
+```php
+<?php
+// Retrieving properties of the first anchor
+$value = $givenNamesSources[0]->getValue(); // string
+$subType = $givenNamesSources[0]->getSubType(); // string
+$signature = $givenNamesSources[0]->getSignature(); // bytes
+$artifactSignature = $givenNamesSources[0]->getArtifactSignature(); // bytes
```

Comment:
We can remove this method it as it does not add value to who integrates.

Reply:
Ok I will remove it. I remember seeing this method in the dot-net SDK, maybe it should be removed there as well @echarrod

Reply:
I thought we were just not exposing .GetArtifactLink()? This method was in the Google Doc, and also in the "Planning the next major release" wiki